### ISSUE TITLE
2 year old gist proof silently stopped working

### ISSUE THREAD

**Issue by timmc** [keybase/keybase-issues#1992]
2 years ago (June 17 2014) I proved my GitHub identity for https://keybase.io/timmc identity with <https://gist.github.com/timmc/d78ac17dcec759742d04>. Today I was [startled](https://github.com/keybase/keybase-issues/issues/1990#issuecomment-179626958) to discover that the proof was no longer considered valid.

I was told in the original email that

> We'll periodically check your proof every few hours. If it ever fails repeatedly, we'll let you know.

but I never received such a notification. Are notifications in fact enabled? Was there a period where they were not? Have requirements for proof gists changed out from under me?

(I've since added a new proof gist.)

**Comment by maxtaco** [comment]
Our logs don't go back that far (we prune them aggressively for your protection), so I can't see when the checks on your original GH proof failed.  But, I can see that your original GH proof successfully last checked at `2016-02-03 23:27:51 ET`.  Did you push the "Retry" button then? 

IT could be there was an outage for your shard on GH and the failure email was swallowed.  But the proof was still cryptographically valid and still appears when we load it today. Furthermore, we've gotten no other similar reports.

**Comment by timmc** [comment]
Yes, I pushed the retry button and got another failure, then filed the aforementioned ticket. However, I did not *at that time* check the gist; maybe there was an outage. It would be nice if the red "gist!" message included a link to the actual gist so I wouldn't have to page back through my gists to find it. (Luckily I am not a heavy gist user, so it was only 2 pages back!)
